File: 06_Monday/prac.py
def prac():
    lst = []
    num = int(input('How many numbers: '))

    for n in range(num):
        numbers = int(input('Enter number '))
        lst.append(numbers)

    snum = lst[0]
    bnum = lst[0]

    for num in lst:
        if num < snum:
            snum = num
        elif num > bnum:
            bnum = num
    print('The answer is {}'.format(bnum - snum))


def more_prac():
    numbers = [10, 20, 0, 10]
    i = 1
    snum = numbers[0]
    bnum = numbers[0]

    while (i < len(numbers)):
        if numbers[i] < snum:
            snum = numbers[i]
        elif numbers[i] > bnum:
            bnum = numbers[i]
        i += 1
    print('The answer is {}'.format(bnum - snum))


def main():
    prac()
    more_prac()


if __name__ == "__main__":
    main()

File: 06_Monday/prac.js
function prac() {
    const lst = [10, 20, 0, 10]

    let snum = lst[0]
    let bnum = lst[0]

    for (var num of lst) {
        if (num < snum) {
            snum = num
        } else if (num > bnum) {
            bnum = num
        }

    }
    console.log(`The answer is ${bnum - snum}`)

}

function diffPrac() {
    const lst = [10, 20, 0, 10]

    let snum = lst[0]
    let bnum = lst[0]

    lst.forEach(function (element) {
        if (element < snum) {
            snum = element
        } else if (element > bnum) {
            bnum = element
        }
    })
    console.log(`The answer is ${bnum - snum}`)
}
prac()
diffPrac()